src/AzureStack/Subscriptions/Subscriptions/Microsoft.AzureStack.Management.Subscriptions.csproj

```diff
@@ -0,0 +1,19 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.AzureStack.Management.Subscriptions</PackageId>
```

Comment:
@bganapa are you planning to create more management clients like these for AzureStack?
What are the differences between Management.Subscription and AzureStack management.subscrption?
Is it just the API version of something else?

Reply:
The subscription definition model in azurestack is slightly different from the azure as we have few extra properties like OfferId  and a field to differentiate whether the subscription is admin or a normal user subscription
We also have APIs to create and delete subscriptions, In azure there are no APIs to create the subscription as it needs to go thru billing (In azure azstack, billing is pluggable, in essense one can create subscription without billing information if admin allows it)

Reply:
@bganapa is anyone from Subscription RP involved in reviewing your subscription API?

Reply:
We reviewed the spec  and the powershell internally with the subscription RP dev

Reply:
@caross to ACK